src/stability_sdk/client.py

```diff
@@ -621,6 +885,28 @@ def process_cli(logger: logging.Logger = None,
         if args.mask_image:
             args.mask_image = Image.open(args.mask_image)
 
+
+        engine_kwargs = {}
+        if args.upscale_engine == 'none':
+            upscale = False
+        elif args.upscale_engine == 'default':
+            upscale = True
+        else:
+            upscale = True
+            upscale_engine = args.upscale_engine
+            engine_kwargs['upscale_engine'] = upscale_engine
+        
+        if args.enhance_engine == 'none':
+            enhance = False
+        elif args.enhance_engine == 'default':
+            enhance = True
+        else:
+            enhance = True
+            enhance_engine = args.enhance_engine
+            engine_kwargs['enhance_engine'] = enhance_engine
+        
+        engine_kwargs['engine'] = args.engine
```

Comment:
This could be condensed down so there's fewer branches and is easier to follow what's going on I think

```suggestion
        engine_kwargs = {'engine': args.engine}

        upscale = args.upscale_engine != 'none'
        if upscale and args.upscale_engine != 'default':
            engine_kwargs['upscale_engine'] = args.upscale_engine

        enhance = args.enhance_engine != 'none'
        if enhance and args.enhance_engine != 'default':
            engine_kwargs['enhance_engine'] = args.enhance_engine
```